packages/discovery-provider/src/tasks/index_payment_router.py

```diff
@@ -255,7 +255,10 @@ def parse_route_transaction_memos(
                     type=RouteTransactionMemoType.recovery, metadata=None
                 )
                 continue
-            if memo.startswith(GEO_MEMO_STRING):
+            if (
+                memo.startswith(GEO_MEMO_STRING)
+                and shared_config["discprov"]["env"] != "dev"
```

Comment:
I think we probably want to more gracefully handle if there isn't geo data even outside of dev - if this throws it makes payment indexing get stuck. Maybe instead of throwing we should log an error and skip the geo info reporting?